src/components/elementView/powerView/powerView.tsx

```diff
@@ -170,6 +170,9 @@ export class PowerView extends React.PureComponent<
           err.response.data.failure_message
         ) {
           this.addErrorToast(err.response.data.failure_message);
+          this.setState({
```

Comment:
Setting the statusLoaded state so that the spinners disappear even if there is a connection error